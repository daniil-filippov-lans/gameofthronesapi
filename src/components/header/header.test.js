import React from 'react';
import Header from './header';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

describe('Testing <Header/>', () => {
    it('Header have rendered correctly', () => {
        const header = shallow(<Header />);
        expect(toJSON(header)).toMatchSnapshot();
    });
});
