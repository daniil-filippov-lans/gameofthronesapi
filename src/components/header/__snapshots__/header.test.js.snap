// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Header/> Header have rendered correctly 1`] = `
<div
  className="header"
>
  <h3
    className="header-title"
  >
    <Link
      to="/"
    >
      Game of Thrones DB
    </Link>
  </h3>
  <ul
    className="header-list"
  >
    <li>
      <Link
        to="/characters/"
      >
        Characters
      </Link>
    </li>
    <li>
      <Link
        to="/houses/"
      >
        Houses
      </Link>
    </li>
    <li>
      <Link
        to="/books/"
      >
        Books
      </Link>
    </li>
  </ul>
</div>
`;
